Chapter2-LinkedLists/2.4-PartitionLL.js

```diff
@@ -3,6 +3,26 @@ Problem:  Write code to partition a linked list around a value x, such that all
 come before all nodes greater than or equal to x.
 */
 
-function partitionLinkedList(x) {
-	
-}
\ No newline at end of file
+/*
+@params{firstElement} - first element in the list
+@params{x} value to partition around
+*/
+
+function partitionLinkedList(firstElement, x) {
+	var prev = null;
+	var curr = firstElement;
+	var headNode = firstElement;
+
+	//if val is less than the partition, set it as the head to move it left
+	while ( prev.next ) {
```

Comment:
since prev is set to `null` on line 12, it looks like this `while` loop will never be entered